.github/copilot-instructions.md

```diff
@@ -128,6 +128,7 @@ Check that these guidelines are followed for new code.
 - Do not declare new `operator<<(ostream& os, type)` overloads, instead prefer to use a `format_to` member function inside `type` as described in
 src/v/base/format_to.h.
 - Prefer using latest C++ features (C++23).
+- All header files (ending in .h) must include the line: #pragma once
```

Comment:
Isn't this going catastropically wrong as this PR shows?

Reply:
Removed in https://github.com/redpanda-data/redpanda/compare/be3d1dc35640f30e45064722645b9824d062d86f..e0a660ba4f6e7c7591325b1ff4ef519bccd8941f.